blameworthy/indexer.go

```diff
@@ -72,30 +72,84 @@ func (history GitHistory) FileBlame(commitHash string, path string) (*BlameResul
 	return &r, nil
 }
 
-func (history GitHistory) findCommit(commitHash string, path string) (File, int, error) {
+// Produces BlameVector for the given commits for the given path.
+// This method differs from FileBlame in that it does not try to compute previous and next commits.
+func (history GitHistory) FileBlameVectorBatch(commits []string, path string) ([]BlameVector, error) {
```

Comment:
I'll see if I can update the PR to address this comment and the last comment (about having blame vector start from a particular commit instead of the beginning of history.)